app/views/layouts/settings.html.slim

```diff
@@ -0,0 +1,20 @@
+doctype
+html
+  head
+    meta charset="utf-8"
```

Comment:
@Gataniel move all tags that inside `<head>` to separated partial `app/views/shared/layout/head`.